transmart-core-db/grails-app/domain/org/transmartproject/db/dataquery/highdim/rnaseq/transcript/DeRnaseqTranscriptAnnot.groovy

```diff
@@ -0,0 +1,34 @@
+package org.transmartproject.db.dataquery.highdim.rnaseq.transcript
+
+import org.transmartproject.db.dataquery.highdim.DeGplInfo
+
+class DeRnaseqTranscriptAnnot {
+
+    String chromosome
+    Long start
+    Long end
+    String refId
+    String transcript
+
+    static belongsTo = [platform: DeGplInfo]
+
+    static mapping = {
+        table schema: 'deapp'
+
+        id generator: "assigned"
```

Comment:
I know that this works, because the column naming in `transmart-data` is consistent with the default naming strategy of Hibernate. But to be consistent with the rest of the codebase, I would use explicit mapping of table name and column names.
E.g., adding `column: 'id'` in this case.